plugins/extractors/bigquery/bigquery.go

```diff
@@ -219,17 +220,27 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) error {
 			return err
 		}
 
+		ch := make(chan *bigquery.Dataset, len(datasets))
```

Comment:
just parallel all the dataset, and use the buffered channel to process the concurrency of the tables